src/coreclr/src/tools/crossgen2/ILCompiler.ReadyToRun/ObjectWriter/RelocationHelper.cs

```diff
@@ -75,19 +75,28 @@ class RelocationHelper
         /// </summary>
         private byte[] _relocationBuffer = new byte[LongestRelocationBytes];
 
+        private List<(int, int)> _paddingToInject;
+
         /// <summary>
         /// Relocation helper stores the output stream and initializes the PE blob builder enumerator.
         /// </summary>
         /// <param name="outputStream">Output stream for the relocated PE file</param>
         /// <param name="peFileBuilder">PE file blob builder</param>
-        public RelocationHelper(Stream outputStream, ulong defaultImageBase, BlobBuilder peFileBuilder)
+        public RelocationHelper(Stream outputStream, ulong defaultImageBase, BlobBuilder peFileBuilder, List<(int, int)> paddingToInject)
         {
+            _paddingToInject = paddingToInject;
```

Comment:
does `_paddingToInject` need to be cloned here since elements are being removed within `CopyRestOfFile`?